apps/installer/includes/functions.sh

```diff
@@ -7,23 +7,29 @@ function inst_configureOS() {
             # If $OSDISTRO is set, use this value (from config.sh)
             if [ ! -z "$OSDISTRO" ]; then
                 DISTRO=$OSDISTRO
-            # If available, use LSB to identify distribution
-            elif [ -f /etc/lsb-release -o -d /etc/lsb-release.d ]; then
-                DISTRO=$(lsb_release -i | cut -d: -f2 | sed s/'^\t'//)
-            # Otherwise, use release info file
-            else
-                DISTRO=$(ls -d /etc/[A-Za-z]*[_-][rv]e[lr]* | grep -v "lsb" | cut -d'/' -f3 | cut -d'-' -f1 | cut -d'_' -f1)
+                RELEASE=$OSRELEASE
+            # Use LSB to identify distribution
+            else 
+                if ! command -v lsb_release &> /dev/null
+                then
+                    echo "Command 'lsb_release' could not be found."
+                    echo "Try installing it using: sudo apt install lsb-release"
+                    exit 1
```

Comment:
why don't we just attempt to install it? This is what we were doing before in the ubuntu distro file. If we just attempt in install here, it wouldn't be needed in the distro file.

Reply:
Yep, it's done that way now!